src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyFilesAndFolders.scala

```diff
@@ -48,16 +33,19 @@ object EasyFilesAndFolders {
 
   def getExistingParent(pathInDataset: String, datasetId: String): Try[Option[String]] = Try {
     val query: PreparedStatement = conn.prepareStatement("SELECT count(pid) FROM easy_folders WHERE path = ?")
-    val parentPath = pathInDataset
+    val subPaths = pathInDataset
```

Comment:
The intent was to get the first existing ancestor. So, maybe the function should be called `getExistingAncestor` and the variable `ancestors` (if you insist on using a temp for this).